### PR TITLE
BLUE-175 Add AJV validation to syncTrieHashes endpoint

### PR DESCRIPTION
**Pull request by tanuj-shardeum**
https://linear.app/shm/issue/GOLD-134/ajv-addition-and-verification-data-check-binary-sync-trie-hashes

### CODE REVIEW

File: src/types/SyncTrieHashesReq.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import { VectorBufferStream } from '../utils/serialization/VectorBufferStream'
 import { TypeIdentifierEnum } from './enum/TypeIdentifierEnum'
+import { verifyPayload } from './ajv/Helpers'
+import { AJVSchemaEnum } from './enum/AJVSchemaEnum'
 
-const cSyncTrieHashesReqVersion = 1
+export const cSyncTrieHashesReqVersion = 1
```
Comment (abdulazeem-tk4vr) [inline src/types/SyncTrieHashesReq.ts]: This variable does not need to be exported. I don't see it being used anywhere outside the file. Please change it.

Comment (kgmyatthu) [reply src/types/SyncTrieHashesReq.ts]: This is resolve

**Review by kgmyatthu — CHANGES_REQUESTED** [review]
Please add unit test for this and the rest look good to me

File: src/types/ajv/SyncTrieHashesReq.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { addSchema } from '../../utils/serialization/SchemaHelpers'
 import { AJVSchemaEnum } from '../enum/AJVSchemaEnum'
 
-const SyncTrieHashesRequestSchema = {
+const schemaBroadcastStateReq = {
```
Comment (tanuj-shardeum) [inline src/types/ajv/SyncTrieHashesReq.ts]: @kgmyatthu I guess you accidentally changed SchemaSyncTrieHashesReq to schemaBroadcastStateReq. I have fixed this as part of the latest commit

**Comment by tanuj-shardeum** [comment]
Added unit test cases

**Pull request merged**